docs/modules/publisher.md

```diff
@@ -10,8 +10,6 @@ It runs on [Azure IoT Edge](https://azure.microsoft.com/en-us/services/iot-edge/
 
 Please use our released containers for OPC Publisher available in the Microsoft Container Registry, rather than building from sources. The easiest way to deploy OPC Publisher is through the [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft_iot.iotedge-opc-publisher).
 
-[<img src="../media/image-20201028141833399.png" style="zoom:50%;" />](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft_iot.iotedge-opc-publisher)
```

Comment:
@marcschier, the image does not seem to show up in documentation on your branch. Can you please have a look ?

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/20900945/144488908-33cfffec-d778-485e-89c4-9c2532b66ff7.png">

Reply:
Sorry, marked the wrong place with the arrow. It should be between first and second lines under `Getting Started`.